open-metadata-implementation/admin-services/docs/user/omag-server-platform-logging.md

```diff
@@ -0,0 +1,49 @@
+
+# Configuring logging options for OMAG server platform
```

Comment:
Add "developer" to the title to distinguish between this logging and the AuditLog

Reply:
I reprhased it to "developer logging" but I still want to disucss different viewpoint on the call today.